### PR TITLE
[device_info] Add device_info_tizen package

### PR DESCRIPTION
**Pull request by bwikbs**

### CODE REVIEW

File: packages/device_info/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+The Tizen implementation of [`device_info`](https://github.com/flutter/plugins/tree/master/packages/device_info).
+
+## Usage
+
+```yaml
+dependencies:
+  device_info_tizen: ^1.0.0
+```
+
+You can import `device_info` in your Dart code:
```
Comment (swift-kim) [inline packages/device_info/README.md:12]: `device_info` → `device_info_tizen`

**Review by swift-kim — APPROVED** [review]

File: packages/device_info/README.md (hunk 1 above)
Comment (bwikbs) [reply packages/device_info/README.md]: Done.

Comment (swift-kim) [reply packages/device_info/README.md]: Typo deviceinfo -> device_info

If you are to upload multiple plugins the README file can be updated later at once. By the way, is this plugin not supported on TV?

Comment (bwikbs) [reply packages/device_info/README.md]: Technically... It was not tested at TV. Currently I haven't got a commercial binary TV device, and the tv emulator doesn't work. I don't know if this is my problem or the embedder problem.

**Comment by swift-kim** [comment]
| Screenshot from 2021-02-04 16-53-47 | Screenshot from 2021-02-04 16-55-32 |
|-|-|

The example app worked on TV (6.0) and wearable (6.0) emulators.

By the way there's no integration test included in this package. Could you add similar one to [device_info_test.dart](https://github.com/flutter/plugins/blob/master/packages/device_info/device_info/example/integration_test/device_info_test.dart)?

**Comment by bwikbs** [comment]
> The example app worked on TV (6.0) and wearable (6.0) emulators. (Why does the TV emulator report its nativeApiVersion as 3.0?)

Um... well? I have no idea why.. as you know TV side is a little bit special...

**Comment by bwikbs** [comment]
> By the way there's no integration test included in this package. Could you add similar one to [device_info_test.dart](https://github.com/flutter/plugins/blob/master/packages/device_info/device_info/example/integration_test/device_info_test.dart)?

done.

File: packages/device_info/tizen/src/device_info_tizen_plugin.cc [hunk 2]
```diff
@@ -0,0 +1,327 @@
+#include "device_info_tizen_plugin.h"
+
+// For getPlatformVersion; remove unless needed for your plugin implementation.
```
Comment (swift-kim) [inline packages/device_info/tizen/src/device_info_tizen_plugin.cc:3]: Please remove this line.

**Review by swift-kim — COMMENTED** [review]
Thanks!

Regarding the native API version, the value is hard-coded in model-config.xml and has never changed since Tizen 3.0 (TV). I think nobody uses the value.

File: packages/device_info/lib/device_info_tizen.dart [hunk 3]
```diff
@@ -0,0 +1,92 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```
Comment (seungsoo47) [inline packages/device_info/lib/device_info_tizen.dart:1]: If this file is for tizen, does not it need to add our copyright?

File: packages/device_info/tizen/src/device_info_tizen_plugin.cc [hunk 4]
```diff
@@ -0,0 +1,326 @@
+#include "device_info_tizen_plugin.h"
```
Comment (seungsoo47) [inline packages/device_info/tizen/src/device_info_tizen_plugin.cc:1]: Please add our copyright and notice.

Comment (bwikbs) [reply packages/device_info/tizen/src/device_info_tizen_plugin.cc:5]: done.

File: packages/device_info/lib/device_info_tizen.dart (hunk 3 above)
Comment (bwikbs) [reply packages/device_info/lib/device_info_tizen.dart:2]: done.

File: packages/device_info/tizen/src/device_info_tizen_plugin.cc (hunk 2 above)
Comment (bwikbs) [reply packages/device_info/tizen/src/device_info_tizen_plugin.cc]: done.

**Review by seungsoo47 — APPROVED** [review]

**Pull request merged**